composer.json

```diff
@@ -1,36 +1,41 @@
 {
     "name": "spryker-eco/after-pay",
+    "type": "library",
     "description": "AfterPay module",
     "license": "MIT",
     "require": {
-        "php": ">=7.1",
-        "spryker/checkout": "^3.0.0 || ^4.2.0 || ^6.0.0",
-        "spryker/currency": "^2.0.0 || ^3.5.0",
-        "spryker/kernel": "^3.0.0",
-        "spryker/locale": "^3.0.0",
+        "php": ">=7.3",
+        "psr/http-message": "^1.0.0",
```

Comment:
Run the following console command to validate the dependencies:
* `vendor/bin/console dev:dependency:fix SprykerEco.AfterPay`.

Reply:
```
No dependency issues found in SprykerEco.AfterPay
```